CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Development
 
+## Fixed
+
+- DroppedFrames always 0
```

Comment:
Please improve the description here for the changelog. We want to be able to copy&paste this into our release notes